tests/acceptance/registries/discover-test.ts

```diff
@@ -0,0 +1,201 @@
+import EngineInstance from '@ember/engine/instance';
+import { currentURL, visit } from '@ember/test-helpers';
+import { setupEngineApplicationTest } from 'ember-osf-web/tests/helpers/engines';
+import { TestContext } from 'ember-test-helpers';
+import { OrderedSet } from 'immutable';
+import $ from 'jquery';
+import { module, test } from 'qunit';
+import { SearchOptions, SearchOrder, SearchResults } from 'registries/services/search';
+import ShareSearch, {
+    ShareRegistration,
+    ShareTermsAggregation,
+    ShareTermsFilter,
+} from 'registries/services/share-search';
+import sinon from 'sinon';
+
+module('Acceptance | Registries | /discover', hooks => {
+    setupEngineApplicationTest(hooks, 'registries');
+
+    hooks.beforeEach(function(this: TestContext) {
+        // const currentUser = server.create('user');
+        server.create('root', { currentUser: null });
+
+        const engine = this.owner.lookup('-engine-instance:registries-registries') as EngineInstance;
+
+        const shareSearch = new ShareSearch();
+
+        engine.register('service:share-search', shareSearch, { instantiate: false });
+        this.owner.register('service:share-search', shareSearch, { instantiate: false });
+    });
+
+    test('visiting /registries/discover', async function(this: TestContext, assert: Assert) {
+        assert.expect(1);
+        const results: SearchResults<ShareRegistration> = {
+            total: 0,
+            results: [],
+            aggregations: {
+                sources: {
+                    buckets: [],
+                },
+            },
+        };
+
+        sinon.stub(this.owner.lookup('service:share-search'), 'registrations').returns(results);
+
+        await visit('/registries/discover');
+
+        assert.equal(currentURL(), '/registries/discover');
```

Comment:
Please add custom messages for all assertions (unless the default message quite obviously describes what the assertion is checking for). Here something like, 'Stays on Registries discover page'.